src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1052,6 +1052,7 @@ export const ComputeModalBase = ({
                     value: runtimeType,
                     onChange: ({ value }) => {
                       setRuntimeType(value)
+                      updateComputeConfig('masterMachineType', getDefaultMachineType(isDataproc(value), getToolFromRuntime(runtimeType)))
```

Comment:
Is the `runtimeType` in `getToolFromRuntime(runtimeType)` intended to be the current runtime type or the new runtime type set with `setRuntimeType(value)`?

Reply:
That's a good catch. I believe it should be value, as you're suggesting. It seemed to work fine locally testing, but I'll make the change and test it out.